README.md

```diff
@@ -147,6 +147,30 @@ In this example, `/old/custom-segment/path` redirects to `/new/path/custom-segme
 }
 ```
 
+**i18n:** Internationalized content can be served based on `accept-language` or `x-country-code` headers.
+
+Imagine a setup with the following files:
+
+```
+- public/
+  - index.html
+  - i18n/
+    - fr/
```

Comment:
Maybe add examples with ALL, and a language & country code combination here.

Reply:
Added a bit, but I want to rely on linking out to Hosting's docs (so we only really have one place to update).